cmake/Modules/OpmDefaults.cmake

```diff
@@ -5,6 +5,8 @@ is_compiler_gcc_compatible ()
 include(TestCXXAcceptsFlag)
 
 macro (opm_defaults opm)
+
+  message("Processing opm_defaults ${opm}")
```

Comment:
Is this leftover debugging code?

Reply:
?????? - this must be a rebase gone astray.